images/toolbox/Containerfile.fedora

```diff
@@ -0,0 +1,8 @@
+FROM registry.fedoraproject.org/fedora:36
+MAINTAINER Shachar Sharon <ssharon@redhat.com>
+
+LABEL org.opencontainers.image.title="Samba Toolbox container"
+LABEL org.opencontainers.image.description="Samba Toolbox container"
+LABEL org.opencontainers.image.vendor="Samba in Kubernetes"
+
+RUN dnf -y install samba-client samba-test
```

Comment:
How come _Containerfile.fedora_ is not shown as rename of previous _Containerfile_?

Reply:
Here its somehow shown as DELETE(_Containerfile_) + ADD(_Containerfile.fedora_). Changes are well and good. I don't want to hold it any longer.